src/mpyl/stages/discovery.py

```diff
@@ -204,7 +204,7 @@ def find_build_set(  # pylint: disable=too-many-arguments, too-many-locals
 
     build_set: dict[Stage, set[ProjectExecution]] = {}
 
-    build_set_file = Path(BUILD_ARTIFACTS_FOLDER) / "build_plan"
+    build_set_file = Path(BUILD_ARTIFACTS_FOLDER) / "build_plan.pickle"
```

Comment:
No problem with this change, just a reminder that we'll need to change [the monorepo workflow](https://github.com/Vandebron/Vandebron/blob/1c8c48a6e861ef33e17e4cb99a29989ae9ed19d0/.github/workflows/mpyl-pull-request.yml#L77) to publish this new filename.

Reply:
I wanted to make it more explicit which file this is now that we have the same file in `.json` format as well